examples/flutter-ota/test/widget_test.dart

```diff
@@ -1,30 +0,0 @@
-// This is a basic Flutter widget test.
-//
-// To perform an interaction with a widget in your test, use the WidgetTester
-// utility in the flutter_test package. For example, you can send tap and scroll
-// gestures. You can also use WidgetTester to find child widgets in the widget
-// tree, read text, and verify that the values of widget properties are correct.
-
-import 'package:flutter/material.dart';
-import 'package:flutter_test/flutter_test.dart';
-
-import 'package:ToDoApp/main.dart';
-
-void main() {
-  testWidgets('Counter increments smoke test', (WidgetTester tester) async {
```

Comment:
Removed as it was going to fail, I didn't spend time on testing the sample app as I don't think it makes too much sense but please feel free to contradict me and if it is the case we can plan to cover it with some test cases